src/select-menu/src/Option.js

```diff
@@ -37,38 +52,47 @@ const Option = memo(
       textProps = selectedProps
     }
 
+    const { className: themedClassName, ...boxProps } = useStyleConfig(
+      'Option',
+      emptyObject,
+      pseudoSelectors,
+      internalStyles
+    )
+
+    const { getRef, ...listBehaviorProps } = useListBehavior({
+      isSelectable,
+      isSelected,
+      disabled,
+      onSelect
+    })
+
+    const callbackRef = useMergedRef(getRef, forwardedRef)
+
     return (
-      <TableRow
-        isSelectable={isSelectable && !disabled}
-        isHighlighted={isHighlighted}
-        onSelect={onSelect}
-        onDeselect={onDeselect}
-        isSelected={isSelected}
+      <Pane
         style={style}
-        display="flex"
-        alignItems="center"
-        borderBottom="muted"
-        {...(isSelected
-          ? { boxShadow: `inset 2px 0 0 ${tokens.selectedOptionColor}` }
-          : {})}
+        className={themedClassName}
+        {...boxProps}
+        {...listBehaviorProps}
         {...rest}
-        ref={ref}
+        ref={callbackRef}
       >
-        <TextTableCell
+        <Text
           height={height}
-          textProps={textProps}
+          display="flex"
+          alignItems="center"
+          {...textProps}
           paddingX={16}
           borderRight={null}
           flex={1}
-          alignSelf="stretch"
```

Comment:
Curious how this visually changes the item

Reply:
seemed to not actually affect things, which is why i removed it